tests/custom-string-parsers.rs

```diff
@@ -217,7 +217,7 @@ fn test_custom_bool() {
             parse(try_from_str = "parse_bool")
         )]
         verbose: bool,
-        #[structopt(short = "t", parse(try_from_str = "parse_bool"))]
+        #[structopt(short = "t", parse(try_from_str = parse_bool))]
         tribool: Option<bool>,
         #[structopt(short = "b", parse(try_from_str = "parse_bool"))]
```

Comment:
As we are in a breaking change version, we can only support the quote less version

Reply:
OK, makes sense.

Reply:
Done